aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,37 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+
+## January 24 - February 11, 2022
+
+### HPE theme breaking change
+**Text-Weak** - To meet accessibility requirements, the color value for `text-weak` is changing from `#BBBBBB` to `#757575` for light mode and from `606B7D` to `8C98AA` for dark mode. A new namespace `text-xweak` is being added with the value `#BBBBBB` for light mode and `#8C98AA` for dark mode.
```

Comment:
#'s for consistency.

And, won't text-xweak receive the old dark color, since #8C98AA is the new one?

```suggestion
**Text-Weak** - To meet accessibility requirements, the color value for `text-weak` is changing from `#BBBBBB` to `#757575` for light mode and from `#606B7D` to `#8C98AA` for dark mode. A new namespace `text-xweak` is being added with the value `#BBBBBB` for light mode and `#606B7D` for dark mode.
```